src/components/app-configure/auth/SocialAuth.vue

```diff
@@ -133,12 +133,20 @@ function isAuthActive(verifier: SocialAuthVerifier) {
     <SettingCard>
       <template #title>Social Auth</template>
       <template #description>
-        Provide easy onboarding for dApp users with familiar social
-        authentication mechanisms. Arcana securely manages public-private key
-        pair for each authenticated user using distributed key generation.
+        Select the social login provider for onboarding app users by specifying
+        authentication verification details. Use the respective social
+        provider's developer console to obtain the authentication verification
+        details such as Client ID, secret, etc. Arcana uses distributed key
+        generation to assign cryptographic key pair to every authenticated user
```

Comment:
`pair` → `pairs`

Reply:
@mmjee removed the entire sentence and shortened it after reviewing it with @shaloo

Reply:
@mmjee thx for the inputs - I will check in docs in case we are using pairs instead of 'pair' somewhere.  For now, in the interest of brevity, we have removed both of these sentences from the Dashboard. jfyi.